charts/k8ssandra/templates/stargate/ingress.yaml

```diff
@@ -1,14 +1,15 @@
-{{- $traefik := .Values.ingress.traefik -}}
-{{- if $traefik.enabled }}
-{{- if $traefik.stargate.enabled }}
+{{- define "k8ssandra.datacenterSize" -}}
+  {{ (index .Values.cassandra.datacenters 0).size }}
+{{- end }}
+{{- $stargateIngress := .Values.stargate.ingress -}}
+{{- if $stargateIngress.enabled }}
 
-{{- if and $traefik.stargate.cassandra.enabled $traefik.cassandra.enabled }}
-  {{- fail "ingress.traefik.stargate.cassandra.enabled and ingress.traefik.cassandra.enabled cannot both be enabled." }}
+{{- if and .Values.ingress.traefik.cassandra.enabled $stargateIngress.cassandra.enabled }}
```

Comment:
Is it safe to assume `.Values.ingress.traefik.cassandra.enabled` will be something like `$cassandra.ingress.enabled` as part of a separate ticket?

Reply:
> Reaper, and Non-Stargate Cassandra will come in later commits to this PR.

I'm going to close this question since you already answered it in the PR description.

Reply:
Yes, part of this PR, coming in later commits.